### PR TITLE
Ethr-did readme updates

### PR DESCRIPTION
**Pull request by localredhead**
- [x] INSERT @ comment A
  - high-level overview; registry, etc. (Scalable identity method for Ethereum addresses)
  - what is onchain/ offchain
  -  what are claims?
- [x] Explain DID method @ comment B
- [x] Explain what the verification includes @ **Verified JWT** section
  - validate the signature
  - an revocation example 
  - check if JWT is formated correctly?
- [x] Provide an example output @ **Verified JWT** section  
  - highlight information about an issuer
  - if the issuer did not sign correctly, what is the response?
- [x] INSERT @ comment E
  - Overview - which type of keys exist - delegate, recovery
  - what is onchain, what is offchain?
  - privacy implications?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This library is intended to use ethereum addresses as fully self managed [Decent
 
 It also lets you sign standards compliant [JSON Web Tokens (JWT)](https://jwt.io) that can be consumed using the [did-jwt](https://github.com/uport-project/did-jwt) library.
 
+Ethr DID provides a scalable identity method for Ethereum addresses that gives any Ethereum address the ability to collect on-chain and off-chain data. Because Ethr DID allows any Ethereum keypair to become an identity, it is more scalable and privacy-preserving than smart contract-based identity methods, like our previous [Proxy Contract](https://github.com/uport-project/uport-identity/blob/develop/docs/reference/proxy.md).
+
```
Comment (localredhead) [inline README.md]: A

File: README.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ This library is intended to use ethereum addresses as fully self managed [Decent
...
 It supports the proposed [Decentralized Identifiers](https://w3c-ccg.github.io/did-spec/) spec from the [W3C Credentials Community Group](https://w3c-ccg.github.io).
 
 The DID method relies on the [ethr-did-registry](https://github.com/uport-project/ethr-did-registry).
```
Comment (localredhead) [inline README.md]: B

File: README.md [hunk 3]
```diff
@@ -130,13 +139,13 @@ const {payload, issuer} = await verifyJWT(helloJWT)
 
 ## Manage Keys
```
Comment (localredhead) [inline README.md]: E

File: README.md [hunk 4]
```diff
@@ -1,15 +1,49 @@
...
+index: 8
+category: "reference"
+type: "content"
+source: "https://github.com/uport-project/ethr-did/blob/develop/README.md"
+---
+
 # ethr DID library
 
-This library is intended to use ethereum addresses as fully self managed [Decentralized Identifiers](https://w3c-ccg.github.io/did-spec/#decentralized-identifiers-dids) (DIDs) and lets you easily create and manage keys for these identities.
+This library is intended to use ethereum addresses as fully self managed [Decentralized Identifiers](https://w3c-ccg.github.io/did-spec/#decentralized-identifiers-dids) (DIDs) and lets you easily create and manage keys for these identities.  It also lets you sign standards compliant [JSON Web Tokens (JWT)](https://jwt.io) that can be consumed using the [did-jwt](https://github.com/uport-project/did-jwt) library.
```
Comment (pelle) [inline README.md]: We should probably mention that this implements ERC 1056 as well

File: README.md [hunk 5]
```diff
@@ -115,7 +149,35 @@ console.log(issuer)
 
 ```
 
-A consuming app can use it directly using [did-jwt](https://github.com/uport-project/did-jwt) for verifying JWTs.
+Example of a verified JWT:
```
Comment (pelle) [inline README.md]: This is a bit confusing as it is the output of verifyJWT. 

Show the JWT itself first and then the payload, explaining that it contains this

**Pull request merged**